DarkCodex/Patches/Patch_ProdigiousTWF.cs

```diff
@@ -20,12 +20,13 @@ public static IEnumerable<CodeInstruction> Transpiler1(IEnumerable<CodeInstructi
             data.Last().Rewind(f => f.IsStloc(7));
 
             // set variable to true, if weapon should be considered light
-            data.InsertAfter(Patch);
+            //data.InsertAfter(PatchTWF);
+			//The above throws an OpCode conversion error. It's ambiguous which "InsertAfter" it is calling so it goes for the first one and not the Delegate one. --Teraunce
```

Comment:
Not sure how that would happen. Perhaps because you set the project to 'preview'?
Btw, you can fix ambiguous issues by using named parameters. Writing "func: " would work.